lib/package.gi

```diff
@@ -1192,6 +1192,9 @@ InstallGlobalFunction( ReadPackage, function( arg )
     if   Length( arg ) = 1 then
       # Guess the package name.
       pos:= Position( arg[1], '/' );
+      if pos = fail then
+        Error(Concatenation(arg[1], " is not a valid filename. Use 'LoadPackage' to load packages"));
+      fi;
```

Comment:
@ChrisJefferson that may be the case when things that are in the system already for a long time do look inconsistent for new users - even although we're consistently saying everywhere that packages are **loaded**, not **read**.

If you think that this is more widespread case and would like to suggest a longer error message, it should suggest to use one of the three cases: 1-argument version, or 2-argument version, or 'LoadPackage'...